airbyte-integrations/connectors/source-clickhouse/src/main/resources/spec.json

```diff
@@ -0,0 +1,38 @@
+{
+  "documentationUrl": "https://docs.airbyte.io/integrations/destinations/clickhouse",
```

Comment:
Question for my own understanding: we spoke about how ClickHouse has different table engines under the hood. Do you think users/Airbyte might benefit from that being exposed? 

Small sanity check that treating Clickhouse as a JDBC source is good enough for our purposes.

Reply:
As discussed over call, we dont need the user to specify the engine in setup wizard cause our approach is not specific to an engine